.editorconfig

```diff
@@ -0,0 +1,27 @@
+# EditorConfig is awesome: http://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+insert_final_newline = true
+
+[*.{js,go,yml,json}]
+charset = utf-8
+
+[*.go]
+indent_style = space
```

Comment:
Golang's linter (go lint), which is the de-facto standard in the community, uses tabs instead of spaces.